src/uproot/interpretation/strings.py

```diff
@@ -152,74 +157,150 @@ def basket_array(
             cursor_offset=cursor_offset,
             library=library,
         )
+        if (
+            isinstance(library, uproot.interpretation.library.Awkward)
+            and byte_offsets is None
+        ):
+            import awkward.forth
```

Comment:
Before this line, call `uproot._extras.import_awkward()` to make sure users without Awkward get a good error message.